README.md

```diff
@@ -1,4 +1,381 @@
-# GrowthBook SDK Flutter.
+# GrowthBook SDK for flutter.
 
-GrowthBook SDK for flutter.
+![](https://docs.growthbook.io/images/hero2.png)
 
+
+
+## Overview
+
+GrowthBook is an open source feature flagging and experimentation platform that makes it easy to adjust what features are shown users, and run A/B tests, without deploying new code. There are two parts to GrowthBook, the GrowthBook Application, and the SDKs which implement this functionality to your code base. This Flutter SDK allows you to use GrowthBook with your Flutter based mobile application.
+
+![](https://camo.githubusercontent.com/b1d9ad56ab51c4ad1417e9a5ad2a8fe63bcc4755e584ec7defef83755c23f923/687474703a2f2f696d672e736869656c64732e696f2f62616467652f706c6174666f726d2d616e64726f69642d3645444238442e7376673f7374796c653d666c6174) ![](https://camo.githubusercontent.com/1fec6f0d044c5e1d73656bfceed9a78fd4121b17e82a2705d2a47f6fd1f0e3e5/687474703a2f2f696d672e736869656c64732e696f2f62616467652f706c6174666f726d2d696f732d4344434443442e7376673f7374796c653d666c6174)
+
+
+
+
+- **Lightweight and fast**
+- **Supports**
+  - **Android**
+  - **iOS**
+  - **Mac**
+  - **Windows**
+- **Use your existing event tracking (GA, Segment, Mixpanel, custom)**
+- **Adjust variation weights and targeting without deploying new code**
+
+
+
+## Installation
+
+1. Add GrowthBook SDK as dependency in your pubspec.yaml file.
+```yaml
+growthbook_sdk_flutter: ^1.0.0
+```
+
+## Integration
+
+Integration is super easy:
+
+1. Create a GrowthBook API key from the GrowthBook App.
+2. At the start of your app,  do SDK Initialization as per below
```

Comment:
Minor text edit: 
Initialize the SDK within your app using the API key, as below. 
or 
Initialize the SDK at the start of your app using the API key, as below.